src/Mvc/Mvc.Core/src/Properties/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<linker>
+  <assembly fullname="Microsoft.AspNetCore.Mvc.Core">
+    <type fullname="Microsoft.AspNetCore.Internal.TrimmingAppContextSwitches">
+      <method signature="System.Boolean get_EnsureJsonTrimmability()" body="stub" value="true" feature="Microsoft.AspNetCore.EnsureJsonTrimmability" featurevalue="true" />
```

Comment:
It's unfortunate that this file needs to be duplicated because of the `assembly` name. :(

Reply:
We can make it public and have it in a single assembly 🤷‍♂️. Do you prefer that? I can open an api proposal.

Reply:
No, I don't think this is worth being public (yet) until we have a strong reason for it.

The duplication is fine for 2. But if we add another one, we should consider a way to share this.